testing/trino-benchto-benchmarks/.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
What uses this file? Shouldn't it be enough to set the required JDK in the POM?

Reply:
POM only impacts Maven, but in the README we tell developers to run java, so we can leverage jenv if they have it.

Reply:
The README should say explicitly which JDK version to use, without assuming how users manage their JDK installations.